### PR TITLE
Bugfixes on subobject synchronizations

### PR DESCRIPTION
**Pull request by rjzondervan**

### CODE REVIEW

File: api/composer.json [hunk 1]
```diff
@@ -13,8 +13,12 @@
         "alcaeus/mongo-php-adapter": "^1.2",
         "api-platform/core": "^2.6",
         "beberlei/doctrineextensions": "^1.3",
+        "common-gateway/brp-bundle": "dev-feature/use-new-installationService",
         "common-gateway/formio-bundle": "dev-main",
-        "commongateway/corebundle": "^1.0.6",
+        "common-gateway/klanten-bundle": "dev-feature/use-new-installationService",
+        "common-gateway/kvk-bundle": "dev-feature/use-new-installationService",
+        "common-gateway/zgw-bundle": "dev-feature/use-new-installationService",
+        "commongateway/corebundle": "1.0.8",
```
[the remark this answers is not in the document]
Comment (rjzondervan) [reply api/composer.json:21]: Chips, nee, ik haal ze eruit

File: api/src/Subscriber/ProxySubscriber.php [hunk 2]
```diff
@@ -70,6 +70,8 @@ public function proxy(RequestEvent $event): void
         unset($headers['x-endpoint']);
         unset($headers['x-method']);
 
+        var_dump($source->getLocation().$endpoint);
```
[the remark this answers is not in the document]
Comment (rjzondervan) [reply api/src/Subscriber/ProxySubscriber.php:73]: Cool, die had ik nog nie gespot, ik fix 'm

File: api/composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (WilcoLouwerse) [reply api/composer.json:17]: I would say we do want "^1.0.12" for coreBundle now

File: api/src/Service/SynchronizationService.php [hunk 3]
```diff
@@ -97,6 +97,11 @@ public function __construct(CallService $callService, EntityManagerInterface $en
         $this->mappingService = $mappingService;
     }
 
+    public function setConfiguration(array $configuration): void
```
Comment (WilcoLouwerse) [inline api/src/Service/SynchronizationService.php:100]: Why this function? where do we use this?

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]
Pls require latest version CoreBundle (with ^)
And one question about a function

**Review by WilcoLouwerse — DISMISSED** [review]

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**